src/rez/shells.py

```diff
@@ -433,7 +472,7 @@ def _create_ex():
                 assert self.rcfile_arg
                 shell_command = "%s %s" % (self.executable, self.rcfile_arg)
             else:
-                shell_command = self.executable
+                shell_command = '"{}"'.format(self.executable)
```

Comment:
I'm curious, is the double quotes required or it could be single quotes? Single quote are safer because they are not evaluated.